FV3/atmos_model.F90

```diff
@@ -629,6 +635,8 @@ subroutine atmos_model_init (Atmos, Time_init, Time, Time_step)
                         IPD_Interstitial, commglobal, mpp_npes(), Init_parm)
 #else
    call IPD_initialize (IPD_Control, IPD_Data, IPD_Diag, IPD_Restart, Init_parm)
+   call populate_diag_manager_controlled_diagnostics(IPD_Diag_diag_manager_controlled, &
```

Comment:
so only in non-CCPP mode, is that on purpose?

Reply:
Ah good catch, thanks!  This was not intentional.  I'll move this outside this `if`-block.